getting-started/templates/systemlink-values.yaml

```diff
@@ -84,6 +84,32 @@ global:
     ## CDN endpoint for telemetry-driven content. NI does not recommend changing this value.
     ##
     cdnEndpoint: "https://web-sdk.ni.com/"
+  ##
+  ## MongoDB Database Configuration.
+  ## Systemlink Enterprise' various services utilize MongoDB databases for data storage. Customers can deploy their own MongoDB instance
+  ## or utilize a managed solution such as MongoDB Atlas. Once provisioned provide the connection details in the values below.
+  ##
+  mongodb:
+    ## Enable or disable installation of per-service MongoDB instances. 
+    #  <ATTENTION> - If you do *not* have your own MongoDB instance set this (`global.mongodb.install`) to `true`.
+    ##
+    install: false 
```

Comment:
If you haven't already, make sure this is called out in the release notes as a breaking change. Otherwise, customers will uninstall their Mongo instances when they try to upgrade.

Reply:
I'm not familiar with this process yet but I've added it to [an Excel sheet ](https://nio365.sharepoint.com/:x:/r/sites/ASWSystemLink/Shared%20Documents/SLE%20Development/SLE%202023-10_Helm%20Chart%20Breaking%20Changes,%20Software%20Behavior%20Changes%20(including%20Priviliges)%20for%20the%20Release%20Notes%20-%20out%20of%20September%202023.xlsx?d=wee495ecc21094d07aa9ec53ffc7df804&csf=1&web=1&e=oANVh0). Please advise if there's more I need to do to get that into the release notes